src/server_manager/messages/en.json

```diff
@@ -108,6 +108,7 @@
   "gcp-firewall-create-3": "Type '0.0.0.0/0' in the 'Source IP ranges' field.",
   "gcp-firewall-create-4": "Select 'Allow all' under 'Protocols and ports'.",
   "gcp-oauth-connect-title": "Sign in or create an account with Google Cloud Platform.",
+  "gcp-project-setup-error": "An error occurred while configuring your Google Cloud project",
```

Comment:
We are using "setup" in the code and "config" in the user text. Could you make the language consistent?

Reply:
OK, done: "An error occurred while setting up your Google Cloud project".